src/main/java/com/atlassian/db/replica/internal/ReplicaStatement.java

```diff
@@ -24,7 +24,7 @@
     private final List<StatementOperation<Statement>> batches = new ArrayList<>();
     private final ReplicaConsistency consistency;
     private final DatabaseCall databaseCall;
-    private boolean isWriteOperation = true;
```

Comment:
https://github.com/atlassian-labs/db-replica/pull/49#discussion_r589301633

Reply:
@wyrzyk 21 hours ago Member
what's the risk of rolling out DualConnection with this change from yours perspective?

We need to use FF for every rollout (deployment may run outside of our working hours and we are expected to observe the rollout). We also should stop to use global kill-switch. So far I have two propositions about how to solve this problem, I'm also open to other solutions.